firebase.json

```diff
@@ -0,0 +1,74 @@
+{
+  "database": {
+    "rules": "database.rules.json"
+  },
+  "firestore": {
+    "rules": "firestore.rules",
+    "indexes": "firestore.indexes.json"
+  },
+  "functions": [
```

Comment:
I think you don't need `functions`, please reduce all emulators to the once you really need.